<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>CropWidget</class>
 <widget class="QWidget" name="CropWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>-1</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item row="0" column="2" colspan="2" >
    <widget class="QLabel" name="lblSize" >
     <property name="text" >
      <string>&amp;Width and Height</string>
     </property>
     <property name="buddy" >
      <cstring>spnW</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="lblStart" >
     <property name="text" >
      <string>&amp;Starting Positon</string>
     </property>
     <property name="buddy" >
      <cstring>spnX</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QSpinBox" name="spnW" />
   </item>
   <item row="1" column="1" >
    <widget class="QSpinBox" name="spnY" />
   </item>
   <item row="1" column="0" >
    <widget class="QSpinBox" name="spnX" />
   </item>
   <item row="1" column="3" >
    <widget class="QSpinBox" name="spnH" />
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0" />
 <pixmapfunction></pixmapfunction>
 <includes/>
 <resources/>
 <connections/>
</ui>
